products/gitlab.md

```diff
@@ -13,7 +13,17 @@ auto:
   # Reference: https://rubular.com/r/mFfxB8FgXXERX4
 -   regex: '^v?(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.(?<patch>0|[1-9]\d*)-ee?$'
     git: https://gitlab.com/gitlab-org/gitlab.git
+# EOL of R = releaseDate(R+2)
+# Support of R = releaseDate(R+1)
+# Since releases are monthly (22nd of every month)
+# This is quite predictable.
 releases:
+-   releaseCycle: "15.7"
+    support: 2023-01-22
+    eol: 2023-03-22
```

Comment:
EOL of R = releaseDate(R+2) = 2023-02-22 ?

Reply:
Corrected to R+3. 

15.7 will go out of support with 15.8 (Jan), but be supported as the previous minor release once 15.9 comes out (Feb), and finally go EOL once 15.10 comes out in March. On March, 15.10 will be actively supported, 15.8, and 15.9 will get security support.